Dockerfile

```
@@ -0,0 +1,17 @@
+FROM debian
+
+RUN apt-get update
+
+RUN apt-get install -y \
+  lua5.2 \
+  liblua5.2-dev \
+  luarocks \
+  git \
+  libssl1.0-dev \
+  make
+
+RUN luarocks install --server=http://luarocks.org/dev luaffi
+ADD Makefile .
+RUN make setup
+
+ADD kong-plugin-template-transformer-0.8.0-2.rockspec .
```

Could we use something like

```suggestion
ADD kong-plugin-template-transformer-*.rockspec .
```

Just so we don't need to change the Dockerfile every release.